2025-04-12 00:54:54,198 INFO    MainThread:14106 [wandb_setup.py:_flush():76] Configure stats pid to 14106
2025-04-12 00:54:54,198 INFO    MainThread:14106 [wandb_setup.py:_flush():76] Loading settings from /home/shruti/.config/wandb/settings
2025-04-12 00:54:54,198 INFO    MainThread:14106 [wandb_setup.py:_flush():76] Loading settings from /home/shruti/workspaces/aerial_gym_ws/src/aerial_gym_simulator/aerial_gym/rl_training/rl_games/wandb/settings
2025-04-12 00:54:54,198 INFO    MainThread:14106 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-04-12 00:54:54,198 INFO    MainThread:14106 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'aerial_gym/rl_training/rl_games/runner.py', 'program': 'runner.py'}
2025-04-12 00:54:54,198 INFO    MainThread:14106 [wandb_init.py:_log_setup():461] Logging user logs to /home/shruti/workspaces/aerial_gym_ws/src/aerial_gym_simulator/aerial_gym/rl_training/rl_games/wandb/run-20250412_005454-ytbe1qny/logs/debug.log
2025-04-12 00:54:54,198 INFO    MainThread:14106 [wandb_init.py:_log_setup():462] Logging internal logs to /home/shruti/workspaces/aerial_gym_ws/src/aerial_gym_simulator/aerial_gym/rl_training/rl_games/wandb/run-20250412_005454-ytbe1qny/logs/debug-internal.log
2025-04-12 00:54:54,198 INFO    MainThread:14106 [wandb_init.py:init():495] calling init triggers
2025-04-12 00:54:54,199 INFO    MainThread:14106 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {'params': {'seed': 10, 'algo': {'name': 'a2c_continuous'}, 'model': {'name': 'continuous_a2c_logstd'}, 'load_checkpoint': False, 'network': {'name': 'actor_critic', 'separate': False, 'space': {'continuous': {'mu_activation': 'None', 'sigma_activation': 'None', 'mu_init': {'name': 'default'}, 'sigma_init': {'name': 'const_initializer', 'val': 0}, 'fixed_sigma': True}}, 'mlp': {'units': [256, 128, 64], 'd2rl': False, 'activation': 'elu', 'initializer': {'name': 'default', 'scale': 2}}}, 'config': {'env_name': 'navigation_task', 'env_config': {'num_envs': 512, 'headless': True, 'use_warp': True}, 'name': 'gen_ppo', 'reward_shaper': {'scale_value': 0.1}, 'normalize_advantage': True, 'gamma': 0.98, 'tau': 0.95, 'ppo': True, 'learning_rate': '1e-4', 'lr_schedule': 'adaptive', 'kl_threshold': 0.016, 'save_best_after': 10, 'score_to_win': 100000, 'grad_norm': 1.0, 'entropy_coef': 0, 'truncate_grads': True, 'e_clip': 0.2, 'clip_value': False, 'num_actors': 512, 'horizon_length': 32, 'minibatch_size': 2048, 'mini_epochs': 4, 'critic_coef': 2, 'normalize_input': True, 'bounds_loss_coef': 0.0001, 'max_epochs': 500, 'normalize_value': True, 'use_diagnostics': True, 'value_bootstrap': True, 'use_smooth_clamp': False, 'player': {'use_vecenv': True}}}}
2025-04-12 00:54:54,199 INFO    MainThread:14106 [wandb_init.py:init():548] starting backend
2025-04-12 00:54:54,199 INFO    MainThread:14106 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-12 00:54:54,211 INFO    MainThread:14106 [backend.py:ensure_launched():217] starting backend process...
2025-04-12 00:54:54,216 INFO    MainThread:14106 [backend.py:ensure_launched():222] started backend process with pid: 14167
2025-04-12 00:54:54,217 INFO    MainThread:14106 [wandb_init.py:init():558] backend started and connected
2025-04-12 00:54:54,221 INFO    MainThread:14106 [wandb_init.py:init():634] updated telemetry
2025-04-12 00:54:54,234 INFO    MainThread:14106 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2025-04-12 00:55:24,235 INFO    MainThread:14106 [interface_shared.py:_communicate_run():326] couldn't get run from backend
2025-04-12 00:55:24,236 ERROR   MainThread:14106 [wandb_init.py:init():674] backend process timed out
2025-04-12 00:55:24,236 ERROR   MainThread:14106 [wandb_init.py:init():685] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2025-04-12 00:55:24,250 ERROR   MainThread:14106 [wandb_init.py:init():1070] error
Traceback (most recent call last):
  File "/home/shruti/miniconda3/envs/safe-rl/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1043, in init
    run = wi.init()
  File "/home/shruti/miniconda3/envs/safe-rl/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 689, in init
    backend.cleanup()
  File "/home/shruti/miniconda3/envs/safe-rl/lib/python3.8/site-packages/wandb/sdk/backend/backend.py", line 246, in cleanup
    self.interface.join()
  File "/home/shruti/miniconda3/envs/safe-rl/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 475, in join
    super().join()
  File "/home/shruti/miniconda3/envs/safe-rl/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 666, in join
    _ = self._communicate_shutdown()
  File "/home/shruti/miniconda3/envs/safe-rl/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 472, in _communicate_shutdown
    _ = self._communicate(record)
  File "/home/shruti/miniconda3/envs/safe-rl/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/shruti/miniconda3/envs/safe-rl/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 231, in _communicate_async
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
